examples/switzerland/PlaceInformation_001_Stop_PlaceRef_Request.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OJP xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.siri.org.uk/siri" xmlns:ojp="http://www.vdv.de/ojp" version="1.1-dev" xsi:schemaLocation="http://www.siri.org.uk/siri ../../OJP.xsd">
+	<OJPRequest>
+		<ServiceRequest>
+			<RequestTimestamp>2020-01-19T12:00:00Z</RequestTimestamp>
+			<RequestorRef>IRMA</RequestorRef>
+			<ojp:OJPPlaceInformationRequest>
```

Comment:
I modified @ue71603 s examples from https://github.com/VDVde/OJP/pull/85/files to make them valid with 1.1-dev (`OJPPlaceInformationRequest` instead of `OJPLocationInformationRequest`, `ojp:Name` instead of `ojp:LocationName`, ...)